fiftyone/core/annotation/validate_label_schemas.py

```
@@ -75,81 +86,95 @@ def validate_label_schemas(
+        fn = _validate_float_int_field_label_schema
+    elif type_ in {foac.FLOAT_LIST, foac.INT_LIST}:
+        fn = _validate_float_int_list_field_label_schema
+    elif type_ == foac.ID:
+        fn = _validate_id_field_label_schema
+    elif type_ == foac.STR:
+        fn = _validate_str_field_label_schema
+    elif type_ == foac.STR_LIST:
+        fn = _validate_str_list_field_label_schema
+
+    if fn is None:
+        raise ValueError(f"unsupported field '{field_name}': {str(field)}")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)